<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaterSleepSchedulerMomButtonsWdgt</class>
 <widget class="QWidget" name="WaterSleepSchedulerMomButtonsWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Selected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbRead">
        <property name="text">
         <string>Read</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
          <normaloff>:/katynko/svg2/dashboard-show.svg</normaloff>:/katynko/svg2/dashboard-show.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbWrite">
        <property name="text">
         <string>Write</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../Matilda-units/gui/image-and-font/res.qrc">
          <normaloff>:/katynko/svg4/dashboard-red.svg</normaloff>:/katynko/svg4/dashboard-red.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>221</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Matilda-units/gui/image-and-font/res.qrc"/>
 </resources>
 <connections/>
</ui>
